### ISSUE TITLE
Enhance Rich Text Box Keyboard Usability

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1692]
A keyboard user cannot access all the functionality available with the rich text box used in UOP (`PageInputBox`). If they can access that functionality, it is not obvious. We should investigate whether it's possible to enhance keyboard usability with `PageInputBox`.
